extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -123,7 +123,7 @@ const INTERACTIVE_EMBEDDED_COMPONENTS_SELECTORS = Object.values(
 ).join(',');
 
 /** @private @const {number} */
-const RESIZE_TIMEOUT_MS = 350;
+const RESIZE_TIMEOUT_MS = 1000;
```

Comment:
The debounce function that checks for a resize event uses this timeout. Before it was too short and wouldn't catch the final size before unlistening. 

I think it's better to have a longer timeout, it would benefit devices with slower connections - where tweets take longer to load, and thus getting the final size.